src/database/schema/ChatSchema.ts

```diff
@@ -175,6 +174,16 @@ class ChatSchema implements BaseDbSchema {
     return Promise.resolve(this.fromDatabaseObject(rows.raw()[0]));
   }
 
+  static async isExists(db: SQLiteDatabase, id: string): Promise<ChatSchema> {
+    const selectQuery = `SELECT A.id
+      FROM ${ChatSchema.getTableName()} A
+      WHERE A.id = ?;`;
+
+    const [{rows}] = await db.executeSql(selectQuery, [id]);
+    if (rows.length === 0) return Promise.resolve(null);
+    return Promise.resolve(this.fromDatabaseObject(rows.raw()[0]));
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new method `isExists` seems to be returning a `ChatSchema` object instead of a boolean value indicating whether the chat exists or not. This could lead to confusion for developers who use this method in the future. Consider changing the return type to a boolean for clarity.
```diff
- static async isExists(db: SQLiteDatabase, id: string): Promise<ChatSchema> {
+ static async isExists(db: SQLiteDatabase, id: string): Promise<boolean> {

  const selectQuery = `SELECT A.id
    FROM ${ChatSchema.getTableName()} A
    WHERE A.id = ?;`;

  const [{rows}] = await db.executeSql(selectQuery, [id]);
- if (rows.length === 0) return Promise.resolve(null);
- return Promise.resolve(this.fromDatabaseObject(rows.raw()[0]));
+ return rows.length > 0;
}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->